skrub/_column_associations.py

```diff
@@ -41,6 +41,26 @@ def column_associations(df):
     -------
     dataframe
         The computed associations.
+
+    Notes
+    -----
+    Cramér's V is a measure of association between two nominal variables,
+    giving a value between 0 and +1 (inclusive).
+    * `Cramer V <https://en.wikipedia.org/wiki/Cramér%27s_V>`_
+
+    Examples
+    --------
+    >>> import numpy as np
+    >>> import pandas as pd
+    >>> import skrub
+    >>> pd.set_option('display.precision', 4)
+    >>> rng = np.random.default_rng()
```

Comment:
Doctests will complain if the output of `df.head()` is not exactly the same every time, so this seed should be set to some value like `np.random.default_rng(33)`

Reply:
add seed